package.json

```diff
@@ -56,8 +58,8 @@
     "webfont-loader": "^1.1.0"
   },
   "devDependencies": {
-    "@material-ui/icons": "latest",
-    "@material-ui/styles": "latest",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/styles": "^4.9.10",
```

Comment:
could you remove these from `devDependencies` now that they are in `dependencies`?
(they don't need to be in both)

Reply:
👍 now done